tests/rpmverify.at

```diff
@@ -631,3 +631,21 @@ runroot rpm -Va --nodeps --nouser --nogroup
 ],
 [])
 AT_CLEANUP
+
+AT_SETUP([test for %verify in %files])
+AT_KEYWORDS([verify])
+AT_CHECK([
+RPMDB_INIT
+
+runroot rpmbuild \
+  --quiet -bb /data/SPECS/test-parseForVerify.spec
```

Comment:
Another uncovered case, nice spotting. The spec name doesn't really fit the general style though - we don't test particular functions, we test functionality. Just use something like verifyfiles.spec.

Reply:
Thank you for your comments. I have made relevant changes. Please review the changes.